packages/shopify-codemod/transforms/utils.js

```diff
@@ -1,4 +1,73 @@
 import j from 'jscodeshift';
+import chalk from 'chalk';
+
+const NEWLINE_REGEX = /\r\n|[\n\v\f\r\x85\u2028\u2029]/;
+const CONTEXT_LINES = 2;
+
+function padLeft(text, length) {
+  return (' '.repeat(length) + text).slice(-length);
+}
+
+function codeContext(node, source, color) {
+  const lines = source.split(NEWLINE_REGEX);
+
+  const {start, end} = node.node.loc;
+  const startLine = start.line - 1;
+  const startColumn = start.column;
+  const endLine = end.line - 1;
+  const endColumn = end.column;
+  const contextStart = Math.max(0, startLine - CONTEXT_LINES);
+  const contextEnd = Math.min(lines.length, endLine + CONTEXT_LINES);
+
+  let result = '';
+  for (let i = contextStart; i <= contextEnd; i++) {
```

Comment:
I'll add more detailed thoughts tonight, but my initial impression is that I'm relying on guesswork to know what's happening in here.  Splitting this into separate calculate/render functions might help.

Test would have been nice, but writing them after the fact doesn't feel useful now that this is working.